lib/beaker/dsl/install_utils/pe_utils.rb

```diff
@@ -28,6 +28,8 @@ def configure_pe_defaults_on( hosts )
           block_on hosts do |host|
             if host[:pe_ver] && aio_version?(host) or (host['type'] && host['type'] =~ /aio/)
               add_aio_defaults_on(host)
+              # provide a sane default here for puppetservice
+              host['puppetservice'] ||= 'pe-puppetserver'
```

Comment:
@anodelman why don't this line (and it's duplicate in `foss_utils.rb`) just get added to the aio defaults themselves (or to `add_aio_defaults_on`) so we eliminate the duplicates?